rust/agama-server/src/network/nm/client.rs

```diff
@@ -173,7 +173,7 @@ impl<'a> NetworkManagerClient<'a> {
                 .build()
                 .await?;
             let flags = proxy.flags().await?;
-            if flags > 0 {
+            if flags >= 8 {
```

Comment:
```suggestion
            # https://networkmanager.dev/docs/api/latest/nm-dbus-types.html#NMSettingsConnectionFlags
            if flags >= 8 {
```

Reply:
Please, beware that the comment should start with `//` instead of `#`.